pandas/util/testing.py

```diff
@@ -1345,10 +1347,15 @@ def assert_frame_equal(left, right, check_dtype=True,
     check_like : bool, default False
         If True, ignore the order of index & columns.
         Note: index labels must match their respective rows
-        (same as in columns) - same labels must be with the same data
+        (same as in columns) - same labels must be with the same data.
     obj : str, default 'DataFrame'
         Specify object name being compared, internally used to show appropriate
-        assertion message
+        assertion message.
+
+    See Also
+    --------
+    assert_series_equal: equivalent method for asserting Series equality
+    DataFrame.equals: check DataFrame equality
```

Comment:
Do you think it makes sense adding a example?

Reply:
I'm not sure an example is particularly useful in this case.

Reply:
We've got users of all levels, and things like comparing dataframes with different types are worth showing with examples.

Besides that, can you add a space before the colons in the see also section, capitalize the first letter of the description, and finish the description with a period? If you generate the html `./doc/make.py html --single pandas.util.testing.assert_frame_equal`, I don't think without the space this is being rendered correcty.

Reply:
Added examples and corrected formatting as mentioned.